changelogs/current.yaml

```diff
@@ -227,6 +227,11 @@ new_features:
 - area: redis
   change: |
     added :ref:`wait_for_warm_on_init <envoy_v3_api_field_config.cluster.v3.Cluster.wait_for_warm_on_init>` support for :ref:`Redis Cluster<arch_overview_redis>`.
+- area: access_log
+  change: |
+    added access log support for ``%STREAM_STATE%`` command operator. Using the command operator will extract a value representing the state of the stream.
+    Access logs at the beggining of the stream will have 'Started' value. Periodic access logs during a living stream, will have 'InProgress' value. Default access log,
```

Comment:
_beginning_.
living stream -> active stream.

We should document clearly when the stream transitions to InProgress - this is only in the code right now, which is difficult to find out.

Reply:
Added documentation to access logs